NEWS.md

```diff
@@ -1,5 +1,34 @@
 # NEWS
 
+## 1.0.1 - 2025-02-21
+
+### Package distribution update
+
+In previous releases, Ubuntu packages were available only through the Groonga
+APT repository. Starting with this release, packages are also distributed via
+packages.groonga.org.
```

Comment:
What is the difference between "the Groonga APT repository" and "package.groonga.org"?

Reply:
fix: [9dd1041](https://github.com/groonga/groonga-nginx/pull/25/commits/9dd1041c02677f2452429d89a428b463eda7eb3d) I added the difference about the dependency package.

Reply:
Does "the Groonga APT repository" mean https://launchpad.net/~groonga/+archive/ubuntu/ppa ?
We should call it as "the Groonga APT repository". We should call https://packages.groonga.org/ as "the Groonga APT repository" because packages.groonga.org is managed by us.

Reply:
I see. So both of them are "the Groonga APT repository".
I tried to rewrite it using specified url. What do you think of it? Or should I call the other ways?

Reply:
Ah, sorry.

> We should call it as "the Groonga APT repository". 

↓

We should NOT call it as "the Groonga APT repository".